aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -66,16 +66,6 @@ pub enum ScriptFunctionCall {
         amount: u64,
     },
 
-    /// Create a proposal with the backing `stake_pool`.
-    /// @param execution_hash Required. This is the hash of the resolution script. When the proposal is resolved,
-    /// only the exact script with matching hash can be successfully executed.
-    AptosGovernanceCreateProposal {
```

Comment:
@davidiw @movekevin Why do I see these changes? I'm fully synced to head. Do we have a test or anything to guarantee this is up-to-date?

Reply:
Can you just revert this change? Something probably went wrong with your head sync/rebase.

Reply:
There is nothing to revert here because those changes were automatically created by the build.

Reply:
Ok I reverted this now but we should chat about how generation of those files is ensured to be up-to-date.